scripts/globals/items/scroll_of_instant_retrace.lua

```diff
@@ -0,0 +1,24 @@
+-----------------------------------------
+-- ID: 5428
+-- Scroll of Instant Retrace
+-- Transports the user to their Allied Nation.
+-----------------------------------------
+require("scripts/globals/teleports")
+require("scripts/globals/status")
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if (target:getCampaignAllegiance() > 0) then
+        result = 0
+    else
+        result = 56
+    end
+    return result
```

Comment:
Slightly fewer lines if you want them:
```lua
if not (target:getCampaignAllegiance() > 0) then
    return 0
else
    return 56
end
```

Reply:
outer parens not needed :P

Reply:
Lua can be odd with `not` at times 💦

Reply:
![not](https://user-images.githubusercontent.com/13112942/97819684-1fe03880-1ca2-11eb-9c3f-87c55aa1c105.png)

Reply:
I changed both item and spell to `if not` statements, and yeah the spell didn't work without the parentheses 😹